test/bundle.test.ts

```diff
@@ -56,26 +56,22 @@ describe.skipIf(process.env.SKIP_BUNDLE_SIZE === 'true' || process.env.ECOSYSTEM
         "@vue/server-renderer",
         "@vue/shared",
         "cookie-es",
-        "debug",
         "defu",
         "destr",
         "devalue",
         "estree-walker",
         "h3",
-        "has-flag",
+        "h3-nightly",
```

Comment:
do we expect this? @pi0

Reply:
Yes. Newly supported in nitro it handles the symlink (they are not two packages). More info: https://github.com/unjs/nitro/pull/1553